### PR TITLE
Improve CI config

### PR DESCRIPTION
**Pull request by szepeviktor**
**Ticket**: #no :(

Neat harmonization only.

### CODE REVIEW

**Comment by szepeviktor** [comment]
Very 🍏 

https://travis-ci.org/github/sitecrafting/greg/builds/738525103

File: phpcs.xml [hunk 1]
```diff
@@ -4,20 +4,17 @@
...
+    <file>./test/integration/</file>
     <exclude-pattern>wp-content/</exclude-pattern>
     <exclude-pattern>wp/</exclude-pattern>
-    <exclude-pattern>vendor/*</exclude-pattern>
-    <exclude-pattern>test/themes</exclude-pattern>
-
-    <!-- If no files or directories are specified on the command line, check all relevant files. -->
-    <file>./lib</file>
+    <exclude-pattern>vendor/</exclude-pattern>
+    <exclude-pattern>test/themes/</exclude-pattern>
```
Comment (acobster) [inline phpcs.xml]: We can remove `test/themes` from this list. That's copy/pasta from the old Conifer codebase.

File: .travis.yml [hunk 2]
```diff
@@ -4,33 +4,36 @@ language: php
...
 
 script:
-  - vendor/bin/phpunit
-  - ./vendor/bin/phpcs --standard=./phpcs.xml test/unit test/integration src
-  - vendor/bin/phpstan analyse
+  - composer run-script test
+  - composer run-script sniff
+  - composer run-script phpstan
+  # TODO
+  #- composer run-script docs:build
```
Comment (acobster) [inline .travis.yml]: This can be removed. I don't think we're going to have a full docs site for this plugin.

**Review by acobster — APPROVED** [review]
Looks good, just a couple vestiges that can be removed.

File: phpcs.xml (hunk 1 above)
Comment (szepeviktor) [reply phpcs.xml]: > That's copy/pasta

:)

kép

Comment (szepeviktor) [reply phpcs.xml]: Done.

File: .travis.yml (hunk 2 above)
Comment (szepeviktor) [reply .travis.yml]: Done.

**Comment by acobster** [comment]
Thanks!

**Comment by szepeviktor** [comment]
You're welcome.

**Pull request merged**